tabular/src/autogluon/tabular/models/tab_transformer/tab_transformer_model.py

```diff
@@ -0,0 +1,469 @@
+""" TabTransformer model """
+import logging
+import os
+import time
+
+import numpy as np
+import pandas as pd
+from autogluon.core.utils.loaders import load_pkl
+from tqdm import tqdm
+
+from .hyperparameters.parameters import get_default_param
+from .hyperparameters.searchspaces import get_default_searchspace
+from ..abstract.abstract_model import AbstractModel
+from ...constants import BINARY, REGRESSION
+from autogluon.core.utils import try_import_torch
+
+logger = logging.getLogger(__name__)
+
+
+class TabTransformerModel(AbstractModel):
```

Comment:
docs needed since this is not currently a publicly known model.

Reply:
Added short doc describing the model at a high level.